desktop/src/app/components/navbar/warnings/modals/convert-field-data-modal.component.ts

```diff
@@ -42,6 +45,17 @@ export class ConvertFieldDataModalComponent {
         if (event.key === 'Escape') this.activeModal.dismiss('cancel');
     }
 
```

Comment:
Code style: There should be always two empty lines between functions

Reply:
What about anonymous functions or rather one-liners? Should there be one or two empty lines between them?

Reply:
One-liners should always be grouped at the top of a class (after the constructor) with a single empty line between them (like e. g. [here](https://github.com/dainst/idai-field/blob/18254d33d1a286b40365c334f7039aa858cb835d/desktop/src/app/components/navbar/warnings/modals/fix-outliers-modal.component.ts#L40-L44)).